uint16 chan1
uint16 chan2
duration max_timeshift